src/utils/handleArticleVotes.js

```diff
@@ -0,0 +1,15 @@
+import { api } from "./axiosCrete";
+
+export const incrementVotes = async (articleId) => {
+  const article = await api.patch(`/articles/${articleId}` , { inc_votes: 1 })

```

Comment:
If the patch fails because of connection issues for example, we should feedback to the user 🙂